zoo-promtail/promtail-config.yml

```
@@ -0,0 +1,26 @@
+server:
```

yamllint

[error] wrong new line character: expected \n

(new-lines)